english/candyland/obamacare/hillary/hillary-appears.md

```diff
@@ -0,0 +1,6 @@
+After passing out from a lack of insulin, you realize that someone is standing over you. It's Hillary Clinton, holding out a cup of grape juice befitted with a straw.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.